django_app/settings.py

```diff
@@ -85,28 +84,30 @@
 
 AUTHENTICATION_BACKENDS = [
     # Needed to login by username in Django admin, regardless of `allauth`
-    'django.contrib.auth.backends.ModelBackend',
-
+    "django.contrib.auth.backends.ModelBackend",
     # `allauth` specific authentication methods, such as login by e-mail
-    'allauth.account.auth_backends.AuthenticationBackend',
+    "allauth.account.auth_backends.AuthenticationBackend",
 ]
 
 # db id of the entry in 'Sites' table
-SITE_ID = 1
-if DEVELOPMENT_MODE:
+if DEVELOPMENT_MODE == "production":
+    SITE_ID = 3
+elif DEVELOPMENT_MODE == "staging":
     SITE_ID = 2
+else:
```

Comment:
No. You must check ALL three cases and if it's something different (like "True" or "False" or whatever) you should throw an exception.

Reply:
Ok